cmd/server-main.go

```diff
@@ -80,9 +80,9 @@ EXAMPLES:
 // Check for updates and print a notification message
 func checkUpdate(mode string) {
 	// Its OK to ignore any errors during getUpdateInfo() here.
-	if older, downloadURL, err := getUpdateInfo(1*time.Second, mode); err == nil {
-		if older > time.Duration(0) {
-			log.Println(colorizeUpdateMessage(downloadURL, older))
+	if currentReleaseTime, latestReleaseTime, downloadURL, err := getUpdateInfo(1*time.Second, mode); err == nil {
+		if updateMsg, isOld := computeUpdateMessage(downloadURL, currentReleaseTime, latestReleaseTime); isOld {
```

Comment:
Could you share why `currentReleaseTime` and `latestReleaseTime` are preferred over `older`?